Tasks/AzurePolicyV0/task.json

```diff
@@ -69,14 +68,14 @@
       "resultTemplate": "{ \"Value\" : \"{{{id}}}\", \"DisplayValue\" : \"{{{name}}}\" }"
     }
   ],
-  "instanceNameFormat": "Azure Policy compliance assessment for $(ResourceGroupName)",
+  "instanceNameFormat": "Azure Policy compliance assessment for $(ConnectedServiceName):$(ResourceGroupName)",
```

Comment:
what will be the name of ResourceGroupName is empty?

Reply:
ConnectedServiceName will be the subscriptionID, if no RG is selected you'll see SubscriptionId: here.